test/srtm.jl

```diff
@@ -0,0 +1,26 @@
+
+using RasterDataSources, URIs, Test, Dates
+using RasterDataSources: rasterpath, zipurl, zipname
+
+@testset "SRTM" begin
+
+    zip_url = URI(scheme = "https", host = "srtm.csi.cgiar.org", path = "/wp-content/uploads/files/srtm_5x5/TIFF/srtm_01_02.zip")
+    tile_index1 = CartesianIndex(2, 1)   # [y, x] order
+    tile_index2 = CartesianIndex(2, 2)   # [y, x] order
+    @test zipurl(SRTM, tile_index = tile_index1) == zip_url
```

Comment:
always use semi colons before keyword arguments `;`

Reply:
`;` only after positional arguments? Or do you want `URI(; scheme = "https", host = ...)` as well?

Reply:
You can do that too if you want, probably BlueStyle does. It just screams "keywords here" at you louder than a comma does